src/librustdoc/html/static/main.js

```diff
@@ -2797,8 +2797,8 @@ function defocusSearchBar() {
 
         var infos = [
             "Prefix searches with a type followed by a colon (e.g., <code>fn:</code>) to \
-             restrict the search to a given type.",
-            "Accepted types are: <code>fn</code>, <code>mod</code>, <code>struct</code>, \
+             restrict the search to a given item kind.",
+            "Accepted item kinds are: <code>fn</code>, <code>mod</code>, <code>struct</code>, \
```

Comment:
nit: we said 'item kind' on the line before so it's clear that's what we're talking about now.
```suggestion
            "Accepted kinds are: <code>fn</code>, <code>mod</code>, <code>struct</code>, \
```